chapters/05-funcoes-de-ordem-superior.md

```diff
@@ -1,12 +1,12 @@
 # Funções de ordem superior
 
-> "Tzu-li e Tzu-ssu estavam se gabando do tamanho dos seus últimos programas. _Duzentas mil linhas_, disse Tzu-li, _sem contar os comentários!_ Tzu-ssu repondeu, _Pss, o meu já é quase um milhão de linhas_. Mestre Yuan-Ma disse, _Meu melhor programa tinha quinhentas linhas_. Ouvindo isso os alunos ficaram admirados."
+> "Tzu-li e Tzu-ssu estavam se gabando do tamanho dos seus últimos programas. 'Duzentas mil linhas sem contar os comentários!', disse Tzu-li. Tzu-ssu respondeu: 'Pssh, o meu já tem quase um milhão de linhas'. Mestre Yuan-Ma disse: 'Meu melhor programa tem quinhentas linhas'. Ouvindo isso, Tzu-li e Tzu-ssu ficaram esclarecidos."
```

Comment:
concordo, por isso que foi feita a troca :)

Original: *"Hearing this, Tzu-li and Tzu-ssu were enlightened."*